app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/History.php

```diff
@@ -211,13 +239,30 @@ public function getItemComment(array $item)
      * @param bool $notified
      * @param \DateTimeInterface $created
      * @param string $comment
+     * @param int $type
      * @return array
      */
     protected function _prepareHistoryItem($label, $notified, $created, $comment = '')
     {
         return ['title' => $label, 'notified' => $notified, 'comment' => $comment, 'created_at' => $created];
     }
 
+    /**
+     * Map history items as array with id and type
+     *
+     * @param int $id
+     * @param string $label
+     * @param bool $notified
+     * @param \DateTimeInterface $created
+     * @param string $comment
+     * @param int $type
+     * @return array
+     */
+    protected function _prepareHistoryItemWithId($id, $label, $notified, $created, $comment = '', $type = 0)
```

Comment:
Please declare argument and return types like this:
```suggestion
    private function prepareHistoryItemWithId(int $id, string $label, bool $notified, \DateTimeInterface $created, string $comment = '', int $type = 0): array
```

Reply:
This is missing the leading `_` of the function name.  But these changes have been committed via commit d4c0ba5680698ac002c5c9907982d81a5be2b7a7